cvat-canvas3d/.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019-2020 Intel Corporation
```

Comment:
I believe should be 2021 without 2019 because this file has been created this year. 
Also, please, pull the latest changes from develop branch.

Reply:
Made changes accordingly.